R/GeospatialFunctions.R

```diff
@@ -0,0 +1,571 @@
+#' TADA_MakeSpatial
+#' 
+#' Transform a Water Quality Portal dataframe into a geospatial {sf} object.
+#' 
+#' @param data A dataframe created by `TADA_DataRetrieval()`.
+#' @param crs The coordinate reference system (CRS) you would like the returned point features to be in. The default is CRS 4326 (WGS84).
+#' 
+#' @return The original TADA Water Quality Portal dataframe but as geospatial {sf} point objects. 
+#' 
+#' @seealso [TADA_DataRetrieval()]
+#' 
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' tada_not_spatial <- TADA_DataRetrieval(startDate = "1995-01-01",
+#'                                        endDate = "1995-12-31",
+#'                                        characteristicName = "pH",
+#'                                        statecode = "SC", 
+#'                                        countycode = "Abbeville",
+#'                                        applyautoclean = TRUE)
+#' 
+#' # make `tada_not_spatial` an {sf} object, projected in crs = 4269 (NAD83)                                 
+#' tada_spatial <- TADA_MakeSpatial(data = tada_not_spatial, crs = 4269)
+#' }
```

Comment:
I'm not sure if this is expected, but all of the examples in the document fail for me if I only run `devtools::load_all()` and don't also load {tidyverse}

Reply:
I had the same issue. Can only run the examples if I also load in {tidyverse}.

Reply:
I think I understand what is happening. There are a few tidyr:: functions that are not explicitly listing the package! Will fix.

Reply:
I pulled the newest commit and that seemed to have fixed my problem